packages/components/input/src/textarea.tsx

```diff
@@ -122,13 +121,26 @@ const Textarea = forwardRef<"textarea", TextAreaProps>(
       />
     );
 
+    const end = useMemo(() => {
+      if (isClearable) {
+        return (
+          <button
+            {...getClearButtonProps({className: "relative block opacity-100 p-0 -m-0 -mr-3 -mt-1"})}
```

Comment:
move the style classes to `getClearButtonProps `. also negative margin shouldn't be used.

Reply:
the negative margin is to align the clear button with the label and scroll bar

Reply:
please remove those negative margins. that is not the correct way to align the clear button with the scrollbar.